pkgs/top-level/all-packages.nix

```diff
@@ -9765,9 +9765,7 @@ with pkgs;
 
   one_gadget = callPackage ../development/tools/misc/one_gadget { };
 
-  oneDNN = callPackage ../development/libraries/oneDNN { };
-
-  oneDNN_2 = callPackage ../development/libraries/oneDNN/2.nix { };
+  oneDNN_2 = callPackage ../by-name/on/oneDNN/2.nix { };
```

Comment:
I think it's better to move `pkgs/by-name/on/oneDNN/2.nix` to `pkgs/by-name/on/oneDNN_2/package.nix` and drop this line?